data/downloads/ir_get_data.m

```diff
@@ -0,0 +1,75 @@
+function [data] = ir_get_data(filename_full, varargin)
+% ir_get_data either returns the data at the specified filename, or grabs
+% if from the data/downloads folder. ir_get_data wil download the data from
+% https://github.com/JeffFessler/MIRTdata/ using websave if needed.
+%
+% Optional inputs:
+%   pick - optional second input to ir_read_mat
+%
+% 2021 - Melissa Haskell, mhask@umich.edu
+
+%% Set defaults and parse variable input arguments
+arg.pick = [];
+arg = vararg_pair(arg, varargin);
+
+
+%% Breakdown full filename into parts
+data_dir = fileparts(which('ir_get_data.m'));
+fn_parts = strsplit(filename_full,data_dir);
+sub_dir_fn = fn_parts{2};
+
+sub_dir_fn_parts = strsplit(sub_dir_fn,'/');
```

Comment:
the "/" might be OS specific.  see if there is some "path split" function.
or maybe strsplit with some "pathsep" that is "/" on unix and something else on pc?